scylla-artifacts.py

```diff
@@ -261,6 +261,11 @@ def run(self):
                          '(see logs for details)' % os.path.basename(pkg))
                 raise InstallPackageError(e_msg)
         process.run('/usr/lib/scylla/scylla_io_setup', shell=True)
+
+        uuid_path = '/etc/scylla.d/housekeeping.uuid'
+        url = 'https://i6a5h9l1kl.execute-api.us-east-1.amazonaws.com/prod/check_version?uu=$id&mark=scylla'
```

Comment:
@kongove, please replace the ' ' with " ".
According to @amnonh, this cause the id to be entered as $id instead of the uuid that it stores.